src/box/memtx_tree.cc

```diff
@@ -314,6 +314,8 @@ struct tree_iterator {
 	enum iterator_type type;
 	struct memtx_tree_key_data<USE_HINT> after_data;
 	struct memtx_tree_key_data<USE_HINT> key_data;
+	/** The amount of tuples to skip after the iterator start. */
+	uint32_t skip;
```

Comment:
Nit: Better name it `offset` - to match the iterator parameter.

Reply:
Done.